test/test_interval_strategy.rb

```diff
@@ -101,5 +101,12 @@
         refute Prop::IntervalStrategy.validate_options!(arg)
       end
     end
+
+    describe "when :threshold is zero" do
```

Comment:
maybe a little background:
```suggestion
    describe "when :threshold is set to zero to disable the prop" do
```

Reply:
It works without breaking anything